src/Auth/Providers/AppleProvider.cs

```diff
@@ -9,7 +9,7 @@ public AppleProvider()
             this.AddScopes(DefaultEmailScope);
         }
 
-        public static AuthCredential GetCredential(string accessToken) => GetCredential(FirebaseProviderType.Apple, accessToken, OAuthCredentialTokenType.AccessToken);
+        public static AuthCredential GetCredential(string Token, OAuthCredentialTokenType TokenType) => GetCredential(FirebaseProviderType.Apple, Token, TokenType);
```

Comment:
improper PascalCasing on parameter names
```suggestion
        public static AuthCredential GetCredential(string token, OAuthCredentialTokenType tokenType) => GetCredential(FirebaseProviderType.Apple, token, tokenType);
```

Reply:
Could we also set the default `OAuthCredentialTokenType` to `AcessToken`? That way it's not a breaking change.